docs/src/index.js

```diff
@@ -9,6 +9,8 @@ import ReactPerf from 'react-addons-perf';
 import { render } from 'react-dom';
 import App from 'docs/src/components/App';
 
+require('core-js');
```

Comment:
I don't think that we should add it to the documentation. The logic is the following, we don't require users to get that polyfill, if we add it to the documentation, we will no longer be able to reproduce users issues.